openmc/filter.py

```diff
@@ -158,8 +187,7 @@ def bins(self, bins):
         bins = np.array(bins)
 
         # If the bin is 0D numpy array, promote to 1D.
-        if bins.shape == ():
-            bins.shape = (1,)
+        bins = np.atleast_1d(bins)
```

Comment:
I don't think you need the `bins = np.array(bins)` above now since `atleast_1d` will give you back an ndarray.